Tasks/Common/azure-arm-rest/azure-arm-app-service-kudu.ts

```diff
@@ -0,0 +1,313 @@
+                console.log(tl.loc("Successfullyupdateddeploymenthistory", response.body.url));
+                return response.body.id;
+            }
+
+            throw response;
+        }
+        catch(error) {
+            throw Error(tl.loc('FailedToUpdateDeploymentHistory', this._getFormattedError(error)));
+        }
+    }
+
+    public async getContinuousJobs(): Promise<Array<WebJob>>{
+        var httpRequest = new webClient.WebRequest();
+        httpRequest.method = 'GET';
+        httpRequest.uri = this._client.getRequestUri(`/api/continuouswebjobs`);
+        try {
+            var response = await this._client.beginRequest(httpRequest);
+            tl.debug(`getContinuousJobs. Data: ${JSON.stringify(response)}`);
+            if(response.statusCode == 200) {
+                return response.body as Array<WebJob>;
+            }
+
+            throw response;
+        }
+        catch(error) {
+            throw Error(tl.loc('FailedToGetContinuousWebJobs', this._getFormattedError(error)))
+        }
+    }
+
+    public async startContinuousWebJob(jobName: string): Promise<WebJob> {
+        tl.debug(`Starting Web job: ${jobName}.`);
+        var httpRequest = new webClient.WebRequest();
+        httpRequest.method = 'POST';
+        httpRequest.uri = this._client.getRequestUri(`/api/continuouswebjobs/${jobName}/start`);
+
+        try {
+            var response = await this._client.beginRequest(httpRequest);
+            tl.debug(`startContinuousWebJob. Data: ${JSON.stringify(response)}`);
+            if(response.statusCode == 200) {
+                return response.body as WebJob;
+            }
+
+            throw response;
+        }
+        catch(error) {
+            throw Error(tl.loc('FailedToStartContinuousWebJob', jobName, this._getFormattedError(error)));
+        }
+    }
+
+    public async stopContinuousWebJob(jobName: string): Promise<WebJob> {
+        tl.debug(`Stopping Web job: ${jobName}.`);
+        var httpRequest = new webClient.WebRequest();
+        httpRequest.method = 'POST';
+        httpRequest.uri = this._client.getRequestUri(`/api/continuouswebjobs/${jobName}/stop`);
+
+        try {
+            var response = await this._client.beginRequest(httpRequest);
+            tl.debug(`stopContinuousWebJob. Data: ${JSON.stringify(response)}`);
+            if(response.statusCode == 200) {
+                return response.body as WebJob;
+            }
+
+            throw response;
+        }
+        catch(error) {
+            throw Error(tl.loc('FailedToStopContinuousWebJob', jobName, this._getFormattedError(error)));
+        }
+    }
+
+    public async installSiteExtension(extensionName: string): Promise<SiteExtension> {
+        console.log(tl.loc("InstallingSiteExtension", extensionName));
+        var httpRequest = new webClient.WebRequest();
+        httpRequest.method = 'GET';
+        httpRequest.uri = this._client.getRequestUri(`/api/siteextensions`);
```

Comment:
Should we use '' quotation here instead of `` as there is not variable to resolve.